src/main/target/SPRACINGH7NANO/target.c

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
This is still missing a Unified Target configuration, see https://github.com/betaflight/betaflight#new-requirements-for-the-submission-of-new-and-updated-targets

Reply:
impossible to create using the instructions linked - https://github.com/betaflight/betaflight/blob/master/unified_targets/docs/CreatingAUnifiedTarget.md as there is no unified target for any H7 MCU yet.